tasks/db_deploy/db_deploy.py

```diff
@@ -31,15 +31,16 @@ def handler(
     """
     Lambda handler for db_deploy. The handler generates the database connection
     configuration information, sets logging handler information and calls the
-    Lambda task function. See the `shared_db.get_configuration(db_connect_info_secret_arn)` function for
-    information on the needed environment variables and parameter store names
+    Lambda task function. See the `shared_db.get_configuration(db_connect_info_secret_arn)`
+    function for information on the needed environment variables and parameter store names
     required by this Lambda.
 
     Args:
         event (Dict): Event dictionary passed by AWS.
         context (object): An object required by AWS Lambda.
     Environment Vars:
-        DB_CONNECT_INFO_SECRET_ARN (string): Secret ARN of the AWS secretsmanager secret for connecting to the database.
+        DB_CONNECT_INFO_SECRET_ARN (string): Secret ARN of the AWS secretsmanager secret
```

Comment:
I would follow the same format you did for other items where it was to long by  spacing under the name like
```
﻿DB_CONNECT_INFO_SECRET_ARN (string): 
﻿    Secret ARN of the AWS secretsmanager secret for connecting to the database.
﻿    See shared_db.py's get_configuration for further details.
﻿```

Reply:
updated